src/WebSdk/Publish/Targets/PublishProfiles/DefaultContainerDeploy.pubxml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <WebPublishMethod>Container</WebPublishMethod>
+    <LastUsedBuildConfiguration>Release</LastUsedBuildConfiguration>
+    <LastUsedPlatform>Any CPU</LastUsedPlatform>
```

Comment:
Do we also need to have a PublishDir that tells where the image will be generated ?

Reply:
A container image _could_ be published to a directory, but it's much more common for them to be pushed to a registry (either local docker engine or some remote resource). I'm not sure that PublishDir would have much meaning for this mechanism.

Reply:
This is a great question and I don't think we have an answer yet. We need to nail down whether we ever produce file artifacts, or only push to a container registry--in which case we'd need the URL for that, which I think we expect to define via project property, not here.